templates/default/gulp-tasks/ssh.js

```diff
@@ -14,8 +14,14 @@ gulp.task('ssh-prod', (cb) => {
   const configured = checkConfiguration(username)
 
   if (configured) {
-    const command = `(cd dist/prod; scp -r ${files} ${username}@${host}:${filepath})`
-    shell.exec(command, cb)
+    const mkdirCommand = `ssh ${username}@${host} mkdir -p ${filepath}`;
+    shell.exec(mkdirCommand)
```

Comment:
When not provided with a callback, `shell.exec` works synchronously.

Reply:
Good to know!